packages/controller/lib/setup/setupRepo.js

```diff
@@ -1,14 +1,13 @@
 'use strict';
 
-const fs = require('fs');
-
 /** @class */
 function Repo(options) {
     const { EXIT_CODES } = require('@iobroker/js-controller-common');
     const { tools }  = require('@iobroker/js-controller-common');
     const axios      = require('axios');
     const ioPackage  = require('../../io-package.json');
     const version    = ioPackage.common.version;
+    const fs         = require('fs');
```

Comment:
guess we also should prefer fs-extra here

Reply:
ONly existsSync, readFileSync are used. Why use fs-extra here?